pkg/plugins/backendplugin/coreplugin/core_plugin.go

```diff
@@ -21,6 +23,15 @@ type corePlugin struct {
 // New returns a new backendplugin.PluginFactoryFunc for creating a core (built-in) backendplugin.Plugin.
 func New(opts backend.ServeOpts) backendplugin.PluginFactoryFunc {
 	return func(pluginID string, logger log.Logger, env []string) (backendplugin.Plugin, error) {
+		for _, e := range env {
```

Comment:
i'm now thinking that this probably isn't going to work since the `GF_PLUGIN` env vars don't identify the plugin 🤔

Reply:
@wbrowne  this works with https://github.com/grafana/grafana/pull/46578, but i am wondering what happens when the next core plugin implements that feature.

in your example, would the env var look like `GF_PLUGIN_FOO_BAR=true`?

this is the first time i've looked at how this works, but i was thinking that the SDK wouldn't be able to enable the feature just for cloudwatch since core plugins run in the same process and share their environment

Reply:
>in your example, would the env var look like GF_PLUGIN_FOO_BAR=true
Exactly.

Ah yes I see the issue now. Yeah, unfortunately that's a limiting factor with the SDK and the nature of how we run core plugins. Will have a think on what could be possible..